src/test/java/cloud/fogbow/ras/core/plugins/interoperability/azure/compute/sdk/AzureVirtualMachineOperationSDKTest.java

```diff
@@ -221,8 +217,7 @@ public void testFindVirtualMachineSizeByNameFail()
     // test case: When calling the findVirtualMachineSizeName method with two virtual machine size name that
     // fits in the requirements, it must verify if It returns the smaller virtual machine size.
     @Test
-    public void testFindVirtualMachineSizeNameSuccessfully()
-            throws NoAvailableResourcesException, UnauthenticatedUserException, UnexpectedException {
+    public void testFindVirtualMachineSizeNameSuccessfully() throws FogbowException {
```

Comment:
The method name changed so It's must be changed in the tests context.
findVirtualMachineSizeName -> **findVirtualMachineSize**
Please, review it in whole test class.

Reply:
Sorry, I thought I checked this before, but it looks like I needed to fix this one.

Reply:
Ok, I'll make this change now.